crm_lead_to_task/views/crm_lead_views.xml

```diff
@@ -6,25 +6,15 @@
         <field name="inherit_id" ref="crm.crm_lead_view_form" />
         <field name="arch" type="xml">
             <field name="stage_id" position="before">
-                <!--Create a Task button (no archive) -->
+                <!--Create a Task action-->
                 <button
-                    name="action_crm_to_task"
+                    name="action_create_task"
                     id="action_create_task_from_lead"
                     type="object"
                     string="Create Task"
                     class="oe_stat_button"
                     groups="project.group_project_user"
```

Comment:
issue: salespersons cannot convert to task if they don't have project permissions.

<img width="2517" height="778" alt="Image" src="https://github.com/user-attachments/assets/7ec853ef-8464-4c77-abb8-16a4e04d5947" />


```suggestion
```

Salespeople should be able to perform the conversion (not the linking).